### PR TITLE
✨ New dashboard summary

### PR DESCRIPTION
**Pull request by JuampiRombola**
image

### CODE REVIEW

**Review by jgalat — DISMISSED** [review]

**Review by jgalat — DISMISSED** [review]

**Review by jgalat — DISMISSED** [review]

File: hooks/useNetAPR.ts [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+
+  const fixedDepositsAPRs = useMemo(() => {
+    if (!accounts.length) return;
+    const _fixedPositions = accounts
+      .flatMap(({ market: { asset }, fixedPositions }) => fixedPositions.map((fp) => ({ ...fp, asset })))
+      .filter(({ borrow }) => !borrow);
+    return _fixedPositions.length
+      ? _fixedPositions.reduce(
+          (acc, { rate, asset, maturity }) => {
+            acc[asset.toLowerCase() + maturity] = rate;
```
Comment (sebipap) [inline hooks/useNetAPR.ts:34]:
```suggestion
            acc[asset.toLowerCase() + maturity] = maturity < now ? 0n : rate;
```

Comment (sebipap) [reply hooks/useNetAPR.ts:34]: look exactly/lib https://github.com/exactly/lib/pull/2/files#:~:text=%7D)%20%3D%3E%20(maturity%20%3E%20timestamp

**Review by sebipap — CHANGES_REQUESTED** [review]
maybe I'm wrong, but we're considering fixed positions that already achieved its maturity as if they are still paying interest.
rest lgtm

File: hooks/useNetAPR.ts (hunk 1 above)
Comment (JuampiRombola) [reply hooks/useNetAPR.ts]: you are right, thanks

**Review by jgalat — APPROVED** [review]

**Review by sebipap — APPROVED** [review]
✅

**Pull request merged**